scripts/ec2cleanup.py

```diff
@@ -218,8 +220,30 @@ def delete_queues(provider_mgmt, excluded_queues, output):
         logger.exception('Exception in %r', delete_queues.__name__)
 
 
+def delete_s3_bucket(provider_mgmt, bucket_name, output):
+    bucket_list = []
+    provider_name = provider_mgmt.kwargs['name']
+    try:
+        for bucket in provider_mgmt.list_s3_bucket():
+            if bucket_name in bucket:
+                deleted_bucket = provider_mgmt.delete_s3_buckets(delete_all_objects=True,
```

Comment:
Q: If you need to remove one bucket at a time and your script accepts only one bucket, what is the reason for making delete_s3_buckets method to accept iterable ?

Reply:
Yes, you are right, it is cross-checking with the all existing buckets for string `smartstate` to match and delete single bucket to delete. I have made `delete_s3_buckets` iterable, if somebody passes more that one bucket for deletion.

Reply:
Updated with required changes.